### DISCUSSION TITLE
Export far-field data to .ZBF format

### DISCUSSION THREAD

**Discussion by Jontza** [NanoComp/meep#2015] [Q&A] [answered]
I am trying to export the far-field data from Meep simulation to Zemax OpticStudio for further analysis.
Is there a way in Meep to export the far-field data to .ZBF file format so that it could be used in OpticStudio?

**Comment by stevengj** [comment, marked as answer]
The PyZDDE package contains a [`writeBeamFile` function](https://github.com/xzos/PyZDDE/blob/379db53489ede96c4f1639da401908a1b30bad4a/pyzdde/zfileutils.py#L473-L571) that looks like it can output this format.
